test/e2e/data/infrastructure-docker/v1.7/clusterclass-quick-start.yaml

```
@@ -42,18 +42,33 @@ spec:
           ref:
             apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
             kind: KubeadmConfigTemplate
-            name: quick-start-default-worker-bootstraptemplate
+            name: quick-start-md-default-worker-bootstraptemplate
         infrastructure:
           ref:
             apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
             kind: DockerMachineTemplate
             name: quick-start-default-worker-machinetemplate
       machineHealthCheck:
         maxUnhealthy: 100%
-        unhealthyConditions:
-        - type: e2e.remediation.condition
-          status: "False"
-          timeout: 20s
+        # We are intentionally not setting the 'unhealthyConditions' here to test that the field is optional.
```

```suggestion
        unhealthyConditions:
        - type: e2e.remediation.condition
          status: "False"
          timeout: 20s
```

In Cluster API 1.7 this field is still mandatory